pvsite_datamodel/sqlmodels.py

```diff
@@ -209,6 +209,33 @@ class SiteSQL(Base, CreatedMixin):
     ml_model: Mapped[Optional[MLModelSQL]] = relationship("MLModelSQL", back_populates="sites")
 
 
+class SiteHistorySQL(Base, CreatedMixin):
+    """Class representing the sites history table.
+
+    Stores a history of changes to sites over time. Uses JSONB so that schema changes to the
+    site table do not affect the history table.
+    """
+
+    __tablename__ = "h_sites"
```

Comment:
shoule this be slightly more verbose, maybe `sites_history`

Reply:
Yeah I looked around and went with `h_` as it seems to be a common convention but happy to update it as you're right `_history` is a bit clearer